crates/wash-cli/src/up/mod.rs

```diff
@@ -711,6 +725,21 @@ fn is_process_running(pid: &str) -> bool {
     }
 }
 
+/// Parses max_execution_time input string to return a duration string.
```

Comment:
perhaps it makes sense to adopt something like https://docs.rs/clap-duration/latest/clap_duration/macro.duration_range_value_parse.html instead of doing manual parsing here?

Reply:
actually, I just found out we can just use https://docs.rs/humantime/latest/humantime/struct.Duration.html directly